ffv1.md

```diff
@@ -1117,28 +1142,36 @@ Inferred to be 0 if not present.
 
 ### sar_num
 
-`sar_num` specifies the `Sample` aspect ratio numerator.  
-Inferred to be 0 if not present.  
-A value of 0 means that aspect ratio is unknown.  
-Encoders MUST write 0 if `Sample` aspect ratio is unknown.  
+`sar_num` specifies the `Sample` aspect ratio numerator.
```

Comment:
My goal with the double space at the end of the line was to avoid the new paragraph (IMO too much space between lines here, and this is not a new paragraph).
This patch looks like it adds paragraphs everywhere, IMO too much.

Reply:
I was trying this as a workaround. With mmark 2.0.7, lines suffixed with double-space create a `<br/>` in the xml output and in xml2rfc version 3 that produces this error
```
xml2rfc --html --v3 draft-ietf-cellar-ffv1-06.xml 
Parsing file draft-ietf-cellar-ffv1-06.xml
draft-ietf-cellar-ffv1-06.xml(1271): Error: Did not expect element br there, at /rfc/middle/section[4]/section[5]/section[8]/t[1]/br[1]
draft-ietf-cellar-ffv1-06.xml: Error: Invalid document before running preptool.
Cannot continue, quitting now
```
as in xml2rfc the `<br>` element is only allowed within table cells.